<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loathsome Bask Drinker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1051</width>
      <height>770</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="control_layout">
         <item row="2" column="8">
          <widget class="QCheckBox" name="repeat_warning">
           <property name="text">
            <string>Repeat</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="interval">
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="7">
          <widget class="QSlider" name="warning_volume">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="warning_volume_l">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QSlider" name="switch_volume">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="warning_time_l">
           <property name="text">
            <string>Early warning time (s) (0 = off)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="warning_time">
           <property name="maximum">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="runbutton">
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="timer_l">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="switch_volume_l">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="interval_l">
           <property name="text">
            <string>Interval (s)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="ws_server_port_l">
           <property name="text">
            <string>Websocket server port</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="websocket_port">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>10666</number>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QPushButton" name="start_ws_server">
           <property name="text">
            <string>Start server</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QLabel" name="ws_server_state_l">
           <property name="text">
            <string>Websocket server not running</string>
           </property>
          </widget>
         </item>
         <item row="3" column="8">
          <widget class="QLabel" name="ws_clients_label">
           <property name="text">
            <string>Clients: 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="logo_l">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/bask-drinker-128.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="switch_sfx_layout_vert">
         <item>
          <widget class="QLabel" name="switch_turn_sound_l">
           <property name="text">
            <string>Switch turn sounds</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="switch_sfx_list"/>
         </item>
         <item>
          <layout class="QGridLayout" name="switch_sfx_layout">
           <item row="0" column="1">
            <widget class="QPushButton" name="remove_all_switch_sfx_button">
             <property name="text">
              <string>Remove all</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="load_switch_sfx_button">
             <property name="text">
              <string>Load sfx</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="remove_switch_sfx_button">
             <property name="text">
              <string>Remove selected</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="warning_sfx_layout_vert">
         <item>
          <widget class="QLabel" name="timer_warning_sound_l">
           <property name="text">
            <string>Timer warning sounds</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="warning_sfx_list"/>
         </item>
         <item>
          <layout class="QGridLayout" name="warning_sfx_layout">
           <item row="0" column="2">
            <widget class="QPushButton" name="remove_warning_sfx_button">
             <property name="text">
              <string>Remove selected</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="load_warning_sfx_button">
             <property name="text">
              <string>Load sfx</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="remove_all_warning_sfx_button">
             <property name="text">
              <string>Remove all</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
